EventFiltering/PWGHF/HFFilter.cxx

```diff
@@ -161,10 +161,10 @@ struct HfFilter { // Main struct for HF triggers
   std::array<std::shared_ptr<TH1>, kNCharmParticles> hBDTScoreNonPrompt{};
 
   // Histogram of TPC postcalibration map for pion and proton
-  TH3F* hMapPionMean;
-  TH3F* hMapPionSigma;
-  TH3F* hMapProtonMean;
-  TH3F* hMapProtonSigma;
+  TH3F* hMapPionMean = new TH3F("hMapPionMean", "Map of TPC pion mean; TPCncls; Pin; Eta", 160, 0., 160, 100, 0., 5, 100, -0.8, 0.8);
+  TH3F* hMapPionSigma = new TH3F("hMapPionSigma", "Map of TPC pion sigma; TPCncls; Pin; Eta", 160, 0., 160, 100, 0., 5, 100, -0.8, 0.8);
+  TH3F* hMapProtonMean = new TH3F("hMapProtonMean", "Map of TPC proton mean; TPCncls; Pin; Eta", 160, 0., 160, 100, 0., 5, 100, -0.8, 0.8);
+  TH3F* hMapProtonSigma = new TH3F("hMapProtonSigma", "Map of TPC proton Sigma; TPCncls; Pin; Eta", 160, 0., 160, 100, 0., 5, 100, -0.8, 0.8);
```

Comment:
You have to initialise with `nullptr`.
What you do here instead is a memory leak, where you create new objects that will be left behind in memory later on when you load the histograms from files with `calibList->FindObject`.